### PR TITLE
Add DPS response for symmetric key registration

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: provisioning/device/src/Models/SymmetricKeyRegistrationResult.cs [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+    /// Registration result returned when using symmetric key attestation.
     /// </summary>
     internal class SymmetricKeyRegistrationResult
     {
-        // this class has no implementation. does the service not send a payload?
+        /// <summary>
+        /// The Id of the enrollment group.
+        /// </summary>
+        [JsonProperty(PropertyName = "enrollmentGroupId")]
+        public string EnrollmentGroupId { get; internal set; }
```
Comment (drwill-ms) [inline provisioning/device/src/Models/SymmetricKeyRegistrationResult.cs:17]: Okay, added, but we don't do anything with it. Does it need to be hooked up somewhere?

Comment (drwill-ms) [reply provisioning/device/src/Models/SymmetricKeyRegistrationResult.cs:17]: FWIW, neither is `X509RegistrationResult.EnrollmentGroupId`. We deserialize it but don't do anything with it.

Should we?

Comment (abhipsaMisra) [reply provisioning/device/src/Models/SymmetricKeyRegistrationResult.cs:17]: Looks like we haven't been exposing all fields in the public facing `DeviceRegistrationResult` (eg. `TpmRegistrationResult`, `X509RegistrationResult` etc.). Adding those in the public facing registration result class should fix it.

Comment (drwill-ms) [reply provisioning/device/src/Models/SymmetricKeyRegistrationResult.cs:17]: Should we make them public get then?

**Review by abhipsaMisra — APPROVED** [review]

**Pull request merged**